<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1191</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="WhiteBoardGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBarCommon">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionOpen"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionSelectAll"/>
  </widget>
  <widget class="QToolBar" name="toolBarZoom">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZoomOriginal"/>
   <addaction name="actionZoomFit"/>
  </widget>
  <widget class="QToolBar" name="toolBarTools">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPen"/>
   <addaction name="actionHighlight"/>
   <addaction name="actionText"/>
   <addaction name="actionPointer"/>
   <addaction name="actionDelete"/>
  </widget>
  <widget class="QToolBar" name="toolBarFont">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBarSettings">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/delete_outline_black_24dp.svg</normaloff>:/icons/icons/delete_outline_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionText">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/text_fields_black_24dp.svg</normaloff>:/icons/icons/text_fields_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
   <property name="toolTip">
    <string>Text</string>
   </property>
  </action>
  <action name="actionHighlight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/drive_file_rename_outline_black_24dp.svg</normaloff>:/icons/icons/drive_file_rename_outline_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Highlight</string>
   </property>
   <property name="toolTip">
    <string>Highlight</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/content_paste_black_24dp.svg</normaloff>:/icons/icons/content_paste_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/select_all_black_24dp.svg</normaloff>:/icons/icons/select_all_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="toolTip">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/content_copy_black_24dp.svg</normaloff>:/icons/icons/content_copy_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/save_black_24dp.svg</normaloff>:/icons/icons/save_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/folder_open_black_24dp.svg</normaloff>:/icons/icons/folder_open_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/clear_black_24dp.svg</normaloff>:/icons/icons/clear_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionZoomOriginal">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/zoom_original_black_24dp.svg</normaloff>:/icons/icons/zoom_original_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom Original</string>
   </property>
   <property name="toolTip">
    <string>Zoom Original</string>
   </property>
  </action>
  <action name="actionZoomFit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/crop_free_black_24dp.svg</normaloff>:/icons/icons/crop_free_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom Fit</string>
   </property>
   <property name="toolTip">
    <string>Zoom Fit</string>
   </property>
  </action>
  <action name="actionPointer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/radio_button_checked_black_24dp.svg</normaloff>:/icons/icons/radio_button_checked_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Pointer</string>
   </property>
   <property name="toolTip">
    <string>Pointer</string>
   </property>
  </action>
  <action name="actionPen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/create_black_24dp.svg</normaloff>:/icons/icons/create_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Pen</string>
   </property>
   <property name="toolTip">
    <string>Pen</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WhiteBoardGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>views/whiteboardgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
